include/eve/concept/vectorizable.hpp

```diff
@@ -106,18 +64,21 @@ namespace eve
   //! @groupheader{Examples}
   //! - `std::uint32_t`
   //================================================================================================
-  template<typename T> concept unsigned_scalar_value          = scalar_value<T> && std::unsigned_integral<T>;
+  template<typename T>
+  concept unsigned_scalar_value = scalar_value<T> && std::unsigned_integral<T>;
```

Comment:
Then it is `arithmetic_scalar_value`? It's cant' be logical? Maybe we should write that.

Reply:
All those concepts have to be scalar_value<T> && ???  for subsumption issue.